mteb/models/jina_models.py

```diff
@@ -0,0 +1,9 @@
+from mteb.model_meta import ModelMeta
+
+paraphrase_multilingual_MiniLM_L12_v2 = ModelMeta(
+    name="jinaai/jina-embeddings-v2-base-en",
+    languages=["eng_Latn"],
+    open_source=True,
+    revision="f84569a8c478c7bb2ea3bc187034859870940deb",
+    release_date="2023-10-30",  # release date of paper
+)
```

Comment:
Oh since Jina works out of the box with ST do we even need to add it? I guess it can already be loaded?

Reply:
Good call! Model loading worked without being added to `mteb/models/__init__.py`. Will remove